package.json

```diff
@@ -31,11 +31,10 @@
   "homepage": "https://github.com/hemerajs/fastify-graceful-shutdown#readme",
   "devDependencies": {
     "fastify": "^4.5.3",
+    "mocha": "^10.0.0",
     "np": "^7.6.2",
     "pre-commit": "^1.2.2",
     "prettier": "^2.2.1",
-    "tap": "^15.2.3",
```

Comment:
tap does not work correctly with this plugin, as it registers its own SIGINT handlers, which crashes plugin on the very first instantiation

Reply:
Looks good